vue/pages/apps/index.html

```diff
@@ -39,10 +39,10 @@
                         </div>
                     </div>
                     <div class="column is-4">
-                        <div class="buttons is-right">
-                            <button title="Collapse all projects" class="button" @click.stop.prevent="collapseProjects()">
+                        <div v-if="view.grid" class="buttons is-right">
+                            <button v-if="showCollapseAll" title="collapse all projects" class="button" @click.stop.prevent="collapseProjects()">
```

Comment:
Out of curiosity, what's the reasoning for making "Collapse" lowercase?

Reply:
For sentences,. I use capitalization and punctuation.  When I am not using sentences for descriptive text, I do not use proper sentence form. This descriptive text could be considered a command, but other titles text throughout the app are not necessarily commands; so, for consistency, I am not capitalizing descriptive text as if they were in the form of sentences. Is there a reason to use capitalization here?

Reply:
<img width="180" alt="Screen Shot 2021-06-11 at 10 50 08 AM" src="https://user-images.githubusercontent.com/1449420/121705657-dc08a980-caa2-11eb-83fa-5c426425cf19.png">
No specific reason, however most websites seem to capitalize the first letter even though it may not always be a complete sentence. See above example from GitHub's own UI.

Reply:
"Add header text" is a full sentence with an implied "you" missing punctuation. Maybe we should make it the standard for the portal to always use full sentences. This would be more descriptive and include full punctuation.